src/courses/models.py

```diff
@@ -200,11 +200,17 @@ def default_end_date():
     return date.today() + timedelta(days=135)
 
 
+def formatted_date():
+    return date.today().strftime('%B-%Y')
```

Comment:
This should not be today(), it should be the first_day (as it currently is).  It might need to be set in a post save (creation) signal

Reply:
Correction.  The field can be left blank, and if it's blank, then __str__ returns MMM-YY, if it's not blank, then __str__ returns name field.

Reply:
Where MMM-YY is form the start date, as it currently does.

Reply:
We don't need to set a default on the name field.

Reply:
ah ok

Reply:
right now i have the listview showing its name. im guessing I also show first day as its name if left emtpy?

Reply:
Good point. Yes.  So just put `__str__` in that column.  i.e. the semester object itself (which will return it's __str__ when cast to a string by the template)

Reply:
ill just commit + push for now

Reply:
oof just commited. ill change it